build/build_queryservice.sh

```diff
@@ -1,13 +1,21 @@
 #!/bin/bash
+# Build script for queryservice
+# Downloads the service-dist.tar file from archiva and caches it 
 set -e
 
 TARBALL="service-$QUERYSERVICE_VERSION-dist.tar.gz"
-TEMP_DIR="$(mktemp -d)"
+TARBALL_SIGNATURE="$TARBALL".md5
 
-wget "https://archiva.wikimedia.org/repository/releases/org/wikidata/query/rdf/service/$QUERYSERVICE_VERSION/$TARBALL" \
-    -O "$TEMP_DIR/$TARBALL"
+MD5_SIGNATURE="$(wget -qO- "https://archiva.wikimedia.org/repository/releases/org/wikidata/query/rdf/service/$QUERYSERVICE_VERSION/$TARBALL_SIGNATURE")"
```

Comment:
We download the md5 sum but don't use it except for assuming that it's stable per `$QUERYSERVICE_VERSION`... IMO we should either drop fetching it here (and use "wbqs-ui-$QUERYSERVICE_VERSION" in the `CACHE_PATH`) or use it to verify the tarball's integrity.

Reply:
Yeah you are right makes more sense to use the version in the path.